client/js/fun.js

```diff
@@ -0,0 +1,9 @@
+var fun = function(input) {
```

Comment:
I deleted the file, that was just to double check the slides to make sure it was running correctly.